src/app/components/Form/FormLabel.tsx

```diff
@@ -16,7 +16,7 @@ export function FormLabel(props: FormLabelProps) {
 	return (
 		<label
 			data-testid="FormLabel"
-			className="flex inline-block text-sm FormLabel transition-colors duration-100 text-theme-neutral-dark"
+			className="flex inline-block mb-2 text-sm FormLabel transition-colors duration-100 text-theme-neutral-dark"
```

Comment:
`FormField` already has `space-y-2`

Reply:
Right, for some reason `SignMessage` form labels where stick to the formfield inputs and I patched it with it. Let me check again now.

Reply:
Correction: not the `SignMessage` but forms like `VerifyMessage` don't appear to have `space-y-2` and the label sticks to the input. But even those who have `space-y-2` no extra space is added (e.g `EditContact`), they still have `0.5rem` margin in between.

Reply:
checked the storybook and it looks ok 👍